roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -19,6 +19,9 @@ tenant_network_types = {{ neutron.tenant_network_type }}
 {% endif %}
 mechanism_drivers = {{ mechanism_drivers|join(',') }}
 
+{% set extension_drivers = ['port_security'] %}
+extension_drivers = {{ extension_drivers|join(',') }}
```

Comment:
Why are we bothering to set a variable here, just to use it one line later? Why not just hardcode the driver name? If we aren't going to hard code it, why wouldn't we specify the driver in the role defaults and just read the variable here?